akka/src/main/scala/zio/actors/akka/AkkaTypedActor.scala

```diff
@@ -0,0 +1,41 @@
+package zio.actors.akka
+
+import akka.actor.typed
+import akka.actor.typed.ActorRef
+import zio._
+
+object AkkaTypedActor {
+
+  /**
+   *
+   * @tparam T behavior type
+   * @return reference to the typed actor in effect that can't fail and requires an akka ActorRef
+   */
+  def makeAkkaProvided[T](): URIO[ActorRef[T], AkkaTypedActorRefLocal[T]] =
+    for {
+      akkaActorRef <- ZIO.environment[typed.ActorRef[T]]
+    } yield new AkkaTypedActorRefLocal[T](akkaActorRef)
+}
+
+private[actors] final class AkkaTypedActorRefLocal[F](akkaActor: typed.ActorRef[F]) extends Serializable {
+
+  /**
+   *
+   * Send message to an akka actor as `fire-and-forget`
+   *
+   * @param fa message
+   * @return lifted unit
+   */
+  def !(fa: F): Task[Unit] =
+    for {
+      p     <- Promise.make[Throwable, Unit]
```

Comment:
Why we're using `Promise` and `.fork` here? I think that `UIO(akkaActor ! fa)` is sufficient here.